oaknational/Oak-Web-Application
# [CUR-1046] Add label to lot picker

## Description

Music year: {owa_music_year}

- Added "Choose a curriculum" label to the lot picker

## How to test

1. Go to https://deploy-preview-3059--oak-web-application.netlify.app/teachers/curriculum
2. You should see the new label on the lot picker
3. On mobile mode you should see an adjusted size for the label as per the Figma designs [here](https://www.figma.com/design/3cz9fb0ObFDLv1c212OtJv/Final-designs---Curriculum-squad?node-id=6916-123465&node-type=instance&m=dev)
4. The label should be across all pages with the lot picker (curriculum home page, curricula pages)

## Screenshots

How it used to look (delete if n/a):
<img width="1194" alt="Screenshot 2024-12-09 at 16 07 16" src="https://github.com/user-attachments/assets/20f9336f-b021-4a88-b021-f76246b5992a">

How it should now look:
<img width="1244" alt="Screenshot 2024-12-09 at 16 09 05" src="https://github.com/user-attachments/assets/ced349fd-c904-4ca8-8040-6394af2c5a2d">
<img width="333" alt="Screenshot 2024-12-09 at 16 09 24" src="https://github.com/user-attachments/assets/c2d9721a-974a-4a9f-9f41-67690247a302">



## Checklist
